src/components/InstructionsDrumMachine.js

```diff
@@ -0,0 +1,207 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import EQ from '../images/EQ.svg';
+import recordingLogo from '../images/recording-page.svg';
+import sequencerLogo from '../images/sequencer.svg';
+
+import { useNavigate } from 'react-router-dom';
+
+export default function InstructionsDrumMachine() {
+  let navigate = useNavigate('');
+  const [firstStepsVisible, setFirstStepsVisible] = useState(false);
+  const [secondStepsVisible, setSecondStepsVisible] = useState(false);
+  const [isNavVisible, setIsNavVisible] = useState(false);
+  const [isPopUpVisible, setIsPopUpVisible] = useState(true);
+  return (
+    <>
+      <PopUp visible={isPopUpVisible}>
+        New? <button onClick={instructionClick}>click for Instructions</button>
+        <button onClick={() => setIsPopUpVisible(false)}>X</button>
+      </PopUp>
+      <InstructionOne visible={firstStepsVisible}>
+        click on <br />
+        <img src={EQ} alt="volume-settings" height="25px" width="25px" />
+        <br />
+        to change the volume
+      </InstructionOne>
+      <InstructionTwo visible={firstStepsVisible}>
+        click on a pad to play a sample
+      </InstructionTwo>
+      <InstructionThree visible={firstStepsVisible}>
+        choose a Drum-Loop an click play to have a background sound
```

Comment:
little typo here: ...and click play... ;)

Reply:
thx